scripts/translations.sh

```diff
@@ -91,4 +117,4 @@ for i in $lang_files; do
 done
 echo "$generalline"
 
-# echo "$1 has language $langcode."
+# echo "$1 has language $langcode."
```

Comment:
Oops?

Reply:
Means it's wrong in OpenGFX too (I copy pasted and restored `-a` for grep)
Oh and now I notice I have a typo in commit message.